packages/api/es/search.js

```diff
@@ -76,7 +76,7 @@ class BaseSearch {
     this.params = params;
     //log.debug('Generated params:', params, logDetails);
 
-    this.size = parseInt((params.limit) ? params.limit : 1, 10);
+    this.size = parseInt((params.limit) ? params.limit : 10, 10);
```

Comment:
We can discuss what we want the default limit to be. 1 log was too few as a single task could put out ~5 different log messages.

Reply:
I'd at least make this a const defined here so we can quickly change it or maybe an env variable.

Reply:
I think env variable is a great idea

Reply:
I think this already exists. If we want to change the limit quickly or temporarily we can use the limit parameter.